Gems/AWSCore/Code/Source/ResourceMapping/AWSResourceMappingManager.cpp

```diff
@@ -244,14 +245,16 @@ namespace AWSCore
 
     bool AWSResourceMappingManager::ValidateJsonDocumentAgainstSchema(const rapidjson::Document& jsonDocument)
     {
-        rapidjson::Document jsonSchemaDocument;
-        if (jsonSchemaDocument.Parse(ResourceMappingJsonSchema).HasParseError())
+        AZ::IO::Path engineRootPath = AZ::IO::PathView(AZ::Utils::GetExecutableDirectory());
```

Comment:
I would name this "executablePath", "engineRoot" is usually where the engine.json file is

Reply:
done